### PR TITLE
Droid Bottom Nav Bar

### PR DESCRIPTION
**Pull request by AliNaqvi97**
Changed from nav drawer to bottom nav bar. Fixed some theming. Renamed Acknowledgements to Info. Removed drawer stuff. Added better transitions for fragments, doesn't reload them every time, shows or hides if already created. Better access to different tabs and also doesn't change Twitter WebView status.

### CODE REVIEW

File: mobile/Client/Client.Droid/MainActivity.cs [hunk 1]
```diff
@@ -1,106 +1,64 @@
...
-using Android.Views;
-using System;
 using Client.Droid.Screens;
 using Android.Gms.Common;
-using Android.Util;
-using Firebase.Iid;
 using Microsoft.AppCenter;
 using Microsoft.AppCenter.Analytics;
 using Microsoft.AppCenter.Crashes;
+using Android.Support.V7.App;
```
Comment (Saadnajmi) [inline mobile/Client/Client.Droid/MainActivity.cs]: using V7 = Android.Support.V7;

to differentiate v4 from v7

File: mobile/Client/Client.Droid/MainActivity.cs [hunk 2]
```diff
@@ -1,106 +1,64 @@
...
-
-        protected override void OnCreate(Bundle savedInstanceState)
-        {
-            base.OnCreate(savedInstanceState);
+	[Activity(Label = "@string/app_name_short", MainLauncher = true, ScreenOrientation = Android.Content.PM.ScreenOrientation.Portrait)]
+	public class MainActivity : AppCompatActivity, BottomNavigationBar.Listeners.IOnTabClickListener
+	{
+		int prevPosition = 0;
+		BottomBar bottomBar;
+		V4.App.Fragment[] Fragments { get; set; }
```
Comment (Saadnajmi) [inline mobile/Client/Client.Droid/MainActivity.cs]: I don't suppose we could use normal fragments and not support fragments?

File: mobile/Client/Client.Droid/Resources/values/style.xml [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-    <item name="android:colorPrimaryDark">@color/primaryDarkColor</item>
-    <!--   theme UI controls like checkboxes and text fields -->
-    <item name="android:colorAccent">@color/secondaryColor</item>
-    <!-- -->
-    <item name="android:windowBackground">@color/backgroundColor</item>
-    <!-- -->
-    <item name="android:navigationBarColor">@color/navigationBarColor</item>
-    <!-- -->
-    <item name="android:textColorPrimary">@color/primaryTextColor</item>
+  <style name="UTColors" parent="Theme.AppCompat">
```
Comment (Saadnajmi) [inline mobile/Client/Client.Droid/Resources/values/style.xml]: Why this change?

File: mobile/Client/Client.Droid/MainActivity.cs (hunk 2 above)
Comment (AliNaqvi97) [reply mobile/Client/Client.Droid/MainActivity.cs]: Uhh these were the ones included in all the other fragments so I went with it.

File: mobile/Client/Client.Droid/MainActivity.cs [hunk 4]
```diff
@@ -1,106 +1,64 @@
...
+
+			bottomBar = BottomBar.Attach(this, savedInstanceState);
+			bottomBar.SetItems(new[] {
+				new BottomBarTab(Resource.Drawable.ic_event_white_24dp, Resource.String.schedule_title),
+				new BottomBarTab(Resource.Drawable.ic_notifications_white_24dp, Resource.String.announcements_title),
+				new BottomBarTab(Resource.Drawable.ic_twitter_white_24dp, Resource.String.twitter_title),
+				new BottomBarTab(Resource.Drawable.ic_info_white_24dp, Resource.String.info_title)
+			});
+
+			bottomBar.MapColorForTab(0, "#bf5700");
```
Comment (Saadnajmi) [inline mobile/Client/Client.Droid/MainActivity.cs]: Can we have these refer to the colors.xml?

File: mobile/Client/Client.Droid/Resources/values/style.xml (hunk 3 above)
Comment (AliNaqvi97) [reply mobile/Client/Client.Droid/Resources/values/style.xml]: Need to use AppCompatActivity with bottom nav bar for some reason. Wouldn't work without it.

File: mobile/Client/Client.Droid/MainActivity.cs (hunk 4 above)
Comment (AliNaqvi97) [reply mobile/Client/Client.Droid/MainActivity.cs]: It takes Android.Graphics.Color for some reason and no colorID from resources so would have to reroute a bit but possible, sure.

File: mobile/Client/Client.Droid/MainActivity.cs (hunk 1 above)
Comment (AliNaqvi97) [reply mobile/Client/Client.Droid/MainActivity.cs]: Not sure what you mean by this?

File: mobile/Client/Client.Droid/MainActivity.cs (hunk 2 above)
Comment (Saadnajmi) [reply mobile/Client/Client.Droid/MainActivity.cs]: We could use FragmentManager, but I feel like I only want to bc I hate seeing the word "support" in code to support older API's I'm not even trying to support

File: mobile/Client/Client.Droid/Resources/values/style.xml (hunk 3 above)
Comment (Saadnajmi) [reply mobile/Client/Client.Droid/Resources/values/style.xml]: Oh. Interesting..

File: mobile/Client/Client.Droid/MainActivity.cs (hunk 2 above)
Comment (AliNaqvi97) [reply mobile/Client/Client.Droid/MainActivity.cs]: Ah makes sense. Eh maybe a last minute thing so it doesn't break for now?

File: mobile/Client/Client.Droid/MainActivity.cs (hunk 4 above)
Comment (Saadnajmi) [reply mobile/Client/Client.Droid/MainActivity.cs]: If not a timesink, then yeah I would appreciate that. I'm looking into how to make the xmls refer to non statis stuff / stuff from another project, if that helps

File: mobile/Client/Client.Droid/Resources/values/style.xml (hunk 3 above)
Comment (AliNaqvi97) [reply mobile/Client/Client.Droid/Resources/values/style.xml]: I believe this is also the reason the app bar breaks now lol

File: mobile/Client/Client.Droid/MainActivity.cs (hunk 1 above)
Comment (Saadnajmi) [reply mobile/Client/Client.Droid/MainActivity.cs]: using V4 = Android.Support.V4;
 using V7 = Android.Support.V7;

Put those as your directives, then in code do stuff like v7.App.blah

Comment (AliNaqvi97) [reply mobile/Client/Client.Droid/MainActivity.cs]: Ooo fancy, didn't know I could do that.
